components/google_contacts/package.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "@pipedream/google_contacts",
+    "version": "0.0.1",
+    "description": "Pipedream Gooogle Contacts Components",
+    "main": "google_contacts.app.mjs",
+    "keywords": [
+        "pipedream",
+        "google_contacts"
+    ],
+    "homepage": "https://pipedream.com/apps/google_contacts",
+    "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+    "license": "MIT",
+    "dependencies": {
+        "@pipedream/platform": "^6.14.13",
```

Got an error trying to install version `^6.14.13`. Running `npm i @pipedream/platform` gave me this version:

```suggestion
        "@pipedream/platform": "^0.10.0",
```